units/patch_intent.go

```diff
@@ -836,6 +821,27 @@ func (j *patchIntentProcessor) buildTriggerPatchDoc(ctx context.Context, patchDo
 	return nil
 }
 
+func (j *patchIntentProcessor) verifyValidAlias(projectId string, patchDoc *patch.Patch) error {
+	alias := j.intent.GetAlias()
+	if alias != "" {
+		validAlias := false
+		aliases, err := model.FindAliasInProjectOrPatchedConfig(projectId, alias, patchDoc.PatchedProjectConfig)
+		if err != nil {
+			return errors.Wrapf(err, "error retrieving aliases for project %s", projectId)
+		}
+		for _, al := range aliases {
+			if al.Alias == alias {
+				validAlias = true
```

Comment:
we can just return nil here now instead of holding onto `validAlias`, and return `errors.Errorf("alias %s is not set on project %s", alias, projectId)` at 842. We can also return immediately at the top if alias == "" so that most of the code doesn't need to be in an if statement